crates/modelardb_server/src/main.rs

```diff
@@ -240,6 +281,46 @@ fn argument_to_local_object_store(argument: &str) -> Result<Arc<dyn ObjectStore>
     Ok(Arc::new(object_store))
 }
 
+/// Retrieve the object store connection info from the ModelarDB manager and use it to connect to
+/// the remote object store. If the connection information could not be retrieved or a connection
+/// could not be established, [`String`] is returned.
+async fn retrieve_manager_object_store(
+    manager_url: String,
```

Comment:
Maybe change the type from `String` to `&str` and remove `to_string()` at the call site.

Reply:
Fixed.